client/components/app.jsx

```diff
@@ -19,26 +21,47 @@ class App extends React.Component {
         this.setState({
           grades: grades
         });
-      });
+      })
+      .catch(err => console.err(err.message));
   }
 
   getAverageGrade() {
     if (this.state.grades.length > 0) {
-      const gradesArray = this.state.grades.slice();
-      const averageArr = [];
-      gradesArray.map(grade => { averageArr.push(grade.grade); });
-      const sum = averageArr.reduce((total, num) => { return total + num; });
-      const average = sum / gradesArray.length;
-      return average;
-    }
+      const gradesArray = this.state.grades;
+      const indivGrades = [];
+      gradesArray.map(grade => {
+        indivGrades.push(grade.grade);
+      });
```

Comment:
If you use `.map()` you shouldn't ignore the new array it returns. In those cases, `forEach` is more appropriate (it doesn't return anything).

so you have two reasonable options here.

```js
      const indivGrades = [];
      gradesArray.forEach(grade => {
        indivGrades.push(grade.grade);
      });
```

```js
const indivGrades = gradesArray.map(grade => grade.grade)
```